spec/services/atmosphere/cloud/vmt_updater_spec.rb

```diff
@@ -154,28 +154,28 @@
     updater.execute
   end
 
-  def open_stack_image(image_id, source_cs, source_uuid)
+  def open_stack_image(image_id, source_t, source_uuid)
     Fog::Compute::OpenStack::Image.new({
       id: image_id,
       name: 'vmt',
       status: 'ACTIVE',
       service: double(list_metadata:
         double(body: {'metadata' => {
-          'source_cs' => source_cs,
+          'source_t' => source_t,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.